packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -86,15 +104,24 @@ export class SidebarComponent extends BaseContainer {
 			// This works better than using CSS pseudo-classes like `:has-text` or `:text-matches` for text
 			// matching.
 			selector = `${ selectors.subheading } >> text="${ subitem }"`;
+			await this._click( selector );
 		}
+	}
 
-		if ( ! selector ) {
-			throw new Error(
-				`Selector is undefined. Check if item or subitem has been specified as argument(s).`
-			);
-		}
-
-		await this._click( selector );
+	/**
+	 * Opens the sidebar into view.
+	 *
+	 * @returns {Promise<void>} No return value.
+	 */
+	async _openSidebar(): Promise< void > {
+		const navbarComponent = await NavbarComponent.Expect( this.page );
+		await navbarComponent.clickMySite();
+		// Layout specifies whether sidebar or main content is being focused at this time.
+		const elementHandle = await this.page.waitForSelector( selectors.layout );
+		await elementHandle.waitForElementState( 'stable' );
+		const classAttributes = ( await elementHandle.getAttribute( 'class' ) ) as string;
+		const isSidebarOpen = classAttributes.includes( 'focus-sidebar' );
+		assert.strictEqual( isSidebarOpen, true );
```

Comment:
```suggestion
		const layoutElement = await this.page.waitForSelector( '.layout.focus-sidebar' );
		await layoutElement.waitForElementState( 'stable' );
```

Not sure if that stability check is necessary though as the `focus-*` class seems to be added after the animation finished.

Reply:
You're right, it was not necessary. I have tweaked your suggestion a little to reuse the defined selector for `.layout` portion, but otherwise your suggestion works without issue.